docroot/sites/default/settings.acquia.php

```diff
@@ -51,7 +51,7 @@
  * Set the HTTP header name which stores real client IP. Harmless if not provided.
  * We use https://www.drupal.org/project/reverse_proxy_header because of https://www.drupal.org/project/drupal/issues/3223280
```

Comment:
Lets revert this comment as well.
```suggestion
 * See https://www.drupal.org/project/reverse_proxy_header
```